File: benchmark/corpus/08_memory/b_log_kept_in_memory.js
// An in-memory event log that grows on every iteration and is never trimmed.
const log = [];
let i = 0;
while (true) {
  log.push({ id: i, text: Array.from({ length: 20 }, (_, k) => String(i * k % 97)).join(" ") }); // DANGER
  i = i + 1;
}
console.log(log.length);

File: benchmark/corpus/08_memory/b_log_kept_in_memory.py
# An in-memory event log that grows on every iteration and is never trimmed.
log = []
i = 0
while True:
    log.append({"id": i, "text": " ".join(str(i * k % 97) for k in range(20))})  # DANGER
    i = i + 1
print(len(log))
